### ISSUE TITLE
Engineering Research -- Add some sort of github CI/CD to monitor 508 compliance

### ISSUE THREAD

**Issue by TomNUSDS** [CDCgov/prime-reportstream#3053]
## Problem statement
We have have really good 508 compliance, but as we add features, we should track if it goes down and catch it quickly.

## What you need to know

Lighthouse has a free accessibility checker built into it that's pretty good and there seems to be github ci/cd support? https://github.com/GoogleChrome/lighthouse-ci

We should use that to verify our 508 compliance stays 100% on all React pages (may be tricky for secure pages requiring login). Maybe there is some cypress or jest support that can be used with mocks?

## Acceptance criteria

- [ ] 

## To do

- [ ]

**Comment by TomNUSDS** [comment]
@sean-usds Thoughts?

**Comment by sean-usds** [comment]
Yeah I like this! Seems easy enough to implement. I'll add it to the backlog

**Comment by supriyaaddagada** [comment]
Link to Github action to get 508 compliance details https://github.com/CDCgov/prime-reportstream/actions/runs/10586463123/job/29335275638

Sample report generated as part of the action https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1724206064266-43634.report.html



NEW

[4:58](https://semanticbits.slack.com/archives/C07JLDNS05C/p1724795913045419)
Link to Github action https://github.com/CDCgov/prime-reportstream/actions/runs/10586463123/job/29335275638

**Comment by devopsmatt** [comment]
Review CI/CD pipelines and see if there are any references to 508 compliance scanning.

**Comment by devopsmatt** [comment]
Already resolved by adding Pa11y to the CI pipeline.  Closing.
